src/pages/Settings/SavedSearch.vue

```diff
@@ -35,32 +39,51 @@
 
 <script>
 import gql from 'graphql-tag';
+import mySavedSearches from '@/graphql/query/mySavedSearches.graphql';
 import SaveSearchItem from '@/components/Settings/SaveSearchItem';
 import WwwPage from '@/components/WwwFrame/WwwPage';
 import TheMyKivaSecondaryMenu from '@/components/WwwFrame/Menus/TheMyKivaSecondaryMenu';
 import KvDefaultWrapper from '@/components/Kv/KvDefaultWrapper';
+import KvSectionModalLoader from '@/components/Kv/KvSectionModalLoader';
 import KvButton from '~/@kiva/kv-components/vue/KvButton';
 import KvGrid from '~/@kiva/kv-components/vue/KvGrid';
 
+const pageQuery = gql`query savedSearchPage {
+	my {
+		savedSearches {
+			totalCount
+		}
+	}
+}`;
+
 export default {
-	name: 'SavedSearchBeta',
+	name: 'SavedSearch',
 	components: {
-		SaveSearchItem,
-		WwwPage,
-		TheMyKivaSecondaryMenu,
 		KvButton,
-		KvGrid,
 		KvDefaultWrapper,
+		KvGrid,
+		KvSectionModalLoader,
+		SaveSearchItem,
+		TheMyKivaSecondaryMenu,
+		WwwPage,
 	},
 	inject: ['apollo', 'cookieStore'],
 	data() {
 		return {
-			totalCount: 0,
-			savedSearches: [],
+			limit: 10,
+			loading: true,
 			offset: 0,
-			limit: 40 // TODO: Limit is 40 because of an error via (totalCount) API
+			savedSearches: [],
+			totalCount: 0,
 		};
 	},
+	apollo: {
```

Comment:
Interesting! How are you able to use this as an option instead of original query initialization on Lines 75-88?

Reply:
This is our global apollo option that is used for the initial fetch of page or component data on the server. This data fetch occurs before the created hook.